### PR TITLE
Add SAML/OIDC comparison by David Chadwick

### PR DESCRIPTION
**Pull request by msporny**
Add a comparison between SAML and OpenID Connect done by @David-Chadwick.

### CODE REVIEW

File: comparison.html [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+      <h1>Comparison of Verificable Credentials with SAML and OIDC</h1>
+
+      <p>
+Existing federated identity management (FIM) systems suffer from a
+number of problems, in particular with regard to their trust model,
+economic models and the privacy of their users.
+      </p>
+
+      <p>
+The FIM trust model is wrong as it requires the Identity Provider (IdP)
```
Comment (dlongley) [inline comparison.html]: Let's strike "is wrong as it". It's a subjective value judgment that detracts from the point.

Comment (David-Chadwick) [reply comparison.html]: Accepted

File: comparison.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+directly on <a href="https://github.com/w3c/vc-data-model/issues/">GitHub</a>,
+or send them to
+<a href="mailto:public-vc-comments@w3.org">public-vc-comments@w3.org</a>
+(<a href="mailto:public-vc-comments-request@w3.org?subject=subscribe">subscribe</a>,
+<a href="https://lists.w3.org/Archives/Public/public-vc-comments/">archives</a>).
+      </p>
+    </section>
+
+    <section>
+      <h1>Comparison of Verificable Credentials with SAML and OIDC</h1>
```
Comment (kimdhamilton) [inline comparison.html]: small typo: "Verificable" -> "Verifiable"

Comment (kimdhamilton) [inline comparison.html]: small typo: "Verificable" -> "Verifiable"

File: comparison.html (hunk 1 above)
Comment (msporny) [reply comparison.html]: Fixed.

File: comparison.html (hunk 2 above)
Comment (msporny) [reply comparison.html]: Fixed.

**Comment by burnburn** [comment]
As a novice to SAML and OIDC I would have no idea that this was a comparison since they are referenced but not mentioned by name.  Rather, this reads to me like one of the most compact explanations of why we are doing the VC work that I have ever read.  I was surprised to find it a separate doc and not part of our spec.

The text itself is wonderful and I am willing to accept it as-is to get something in place, but I do have the two concerns above.

**Pull request merged**